### PR TITLE
#157842572 Deploy Wger to Heroku

### PR DESCRIPTION
**Pull request by tibetegya**
### What does this PR do?
Adds configuration settings for heroku deployment.
#### Description of Task to be completed?
1. Add Procfile for heroku deployment
2. Add whitenoise, gunicorn, psycopg2 dependencies to requirements.txt
3. Add whitenoise instance to wsgi
4. Add database configurations to settings global
#### How should this be manually tested?
1. Run the commands below
```
$ git clone https://github.com/andela/wg-rogue-one.git

$ git checkout ch-heroku-developyment-157842572

```
follow the setup instructions in [README.rst](https://github.com/andela/wg-rogue-one/blob/ch-heroku-deployment-157842572/README.rst)
2. Run the app with `python manage.py runserver`
3. To run tests use `python manage.py test`

#### Any background context you want to provide?
The bower components dependencies need to have specific versions in order run the UI appropriately.  
#### What are the relevant pivotal tracker stories?
#157842572

### CODE REVIEW

File: wger/settings_global.py [hunk 1]
```diff
@@ -91,12 +91,12 @@
 # added list of external libraries to be installed by bower
 BOWER_INSTALLED_APPS = (
     'jquery#2.1.x',
-    'bootstrap',
+    'bootstrap#3',
     'd3',
-    'shariff',
+    'shariff#1.24.1',
```
Comment (dondrzzy) [inline wger/settings_global.py:99]: Why did you choose this version?

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import os
+import warnings
+from wger.settings_global import *
+
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
```
Comment (dondrzzy) [inline settings.py]: For production, please set this to False.

**Review by dondrzzy — COMMENTED** [review]
For production, please set this to False.

File: wger/settings_global.py (hunk 1 above)
Comment (tibetegya) [reply wger/settings_global.py:99]: i back dated the version to 2 years ago which i the one they used for this project it creates the build folder which is needed

**Pull request merged**